### PR TITLE
stolostron/backlog#27129

### PR DESCRIPTION
**Pull request by oafischer**
stolostron/backlog#27129

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -988,11 +988,47 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
...
 
 [#imported-managed-clusters-pending-import]
-=== Imported managed clusters show a Pending Import status
+=== Imported managed clusters show a _Pending Import_ status
 //2.7:26797
 
 Managed clusters that are manually imported on the primary hub cluster show a `Pending Import` status when the activation data is restored on the passive hub cluster. For more information, see link:../backup_restore/manage_backup_restore.adoc#auto-connect-clusters-msa[Automatically connecting clusters by using a Managed Service Account (Technology Preview)].
 
+[#appliedmanifestwork-not-removed-after-restoring]
+=== The _appliedmanifestwork_ is not removed after restoring the hub cluster
```
Comment (birsanv) [inline release_notes/known_issues.adoc]: I think this should say : The _appliedmanifestwork_ is not removed **from the managed cluster** after restoring the hub cluster
( the reason : the issue happens on the managed cluster, not on the new, restored, hub cluster

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -988,11 +988,47 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
...
+//2.7:27129
+
+If the placement rule for an application subscription is not a fixed cluster set, the `appliedmanifestwork` from the old hub cluster remains on the new hub cluster after restoring.
```
Comment (birsanv) [inline release_notes/known_issues.adoc]: same comment here as above , should be 

If the placement rule for an application subscription is not a fixed cluster set, when the hub data is restored on the new hub, the `appliedmanifestwork` is not removed **from the managed clusters** matching this placement rule.

**Review by birsanv — CHANGES_REQUESTED** [review]

**Comment by oafischer** [comment]
Thanks, implemented your suggestions @birsanv!

**Review by birsanv — APPROVED** [review]
lgtm

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -988,11 +988,47 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
 - It is possible that a `ManagedServiceAccount` secret might not be included in a backup if the backup schedule runs before the backup label is set on the `ManagedServiceAccount` secret. `ManagedServiceAccount` secrets don't have the cluster `open-cluster-management.io/backup` label set on creation. For this reason, the backup controller regularly searches for `ManagedServiceAccount` secrets under the managed cluster's namespaces, and adds the backup label if not found.
 
 [#imported-managed-clusters-pending-import]
-=== Imported managed clusters show a Pending Import status
+=== Imported managed clusters show a _Pending Import_ status
 //2.7:26797
 
 Managed clusters that are manually imported on the primary hub cluster show a `Pending Import` status when the activation data is restored on the passive hub cluster. For more information, see link:../backup_restore/manage_backup_restore.adoc#auto-connect-clusters-msa[Automatically connecting clusters by using a Managed Service Account (Technology Preview)].
 
+[#appliedmanifestwork-not-removed-after-restoring]
```
Comment (swopebe) [inline release_notes/known_issues.adoc]: I would shorten this anchor to something easy to link to if needed--making these as simple as possible:

#appliedmanifestwork-not-removed (or something--as with file names, we want to keep concise and simple when and where we can)

File: release_notes/known_issues.adoc [hunk 4]
```diff
@@ -988,11 +988,47 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
...
+=== The _appliedmanifestwork_ is not removed from managed clusters after restoring the hub cluster
+//2.7:27129
+
+The `appliedmanifestwork` is not removed from managed clusters that have a placement rule for an application subscription that is not a fixed cluster set when the hub cluster data is restored on the new hub cluster.
+
```
Comment (swopebe) [inline release_notes/known_issues.adoc:1001]: I read this and thought: huh 

Slightly hard to follow, run on sentence. I would offer an edit but in this case, I am not sure how to fix it bc I don't understand it.

File: release_notes/known_issues.adoc [hunk 5]
```diff
@@ -988,11 +988,47 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
...
+[source,yaml]
+----
+spec:
+  clusterReplicas: 1
+  clusterSelector:
+    matchLabels:
+      environment: dev
+----
+
+This results in the application being orphaned when the managed cluster is detached from the restored hub cluster.
```
Comment (swopebe) [inline release_notes/known_issues.adoc]: As a result, the application is orphaned...

File: release_notes/known_issues.adoc [hunk 6]
```diff
@@ -988,11 +988,47 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
...
+    matchLabels:
+      environment: dev
+----
+
+You can also delete the remaining `appliedmanifestwork` manually by running the folowing command:
+
+----
+oc delete appliedmanifestwork <the-left-appliedmanifestwork-name>
+----
+
```
Comment (swopebe) [inline release_notes/known_issues.adoc:1030]: the rest looks good.

**Review by swopebe — CHANGES_REQUESTED** [review]
I got lost a bit on the issue explanation

File: release_notes/known_issues.adoc (hunk 4 above)
Comment (oafischer) [reply release_notes/known_issues.adoc:1001]: Yeah was thinking about that too and hoping a peer review might give me some ideas, but I think I just came up with something that might help. Just updated it.

**Comment by oafischer** [comment]
Moved it around a bit @swopebe, is it better now? Open to suggestions, not an easy one to fix.

File: release_notes/known_issues.adoc [hunk 7]
```diff
@@ -988,11 +988,46 @@ When a cluster is imported on the hub cluster, the `apiserver` is only set up au
...
+=== Imported managed clusters show a _Pending Import_ status
 //2.7:26797
 
 Managed clusters that are manually imported on the primary hub cluster show a `Pending Import` status when the activation data is restored on the passive hub cluster. For more information, see link:../backup_restore/manage_backup_restore.adoc#auto-connect-clusters-msa[Automatically connecting clusters by using a Managed Service Account (Technology Preview)].
 
+[#appliedmanifestwork-not-removed]
+=== The _appliedmanifestwork_ is not removed from managed clusters after restoring the hub cluster
+//2.7:27129
+
+When the hub cluster data is restored on the new hub cluster, the `appliedmanifestwork` is not removed from managed clusters that have a placement rule for an application subscription that is not a fixed cluster set.
```
Comment (birsanv) [inline release_notes/known_issues.adoc:1000]: I agree with @swopebe that the initial sentence was hard to read. I was about to give a suggestion that looks just like yours Oliver : start with `When the hub cluster data is restored on the new hub cluster` .. looks good to me as it is now
